src/components/Modals/RegisterModal/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
Sugestão: como a gente não vai usar tudo que vem do `React`, é interessante a gente puxar só o que iremos usar. Nesse caso, indico fazer a `descontrução do objeto` e pegar somente o necessário.

```suggestion
import { useState } from 'react';
```